test/libsolidity/semanticTests/various/iszero_bnot_correct.sol

```diff
@@ -15,5 +15,7 @@ contract C {
     }
 }
 
+// ====
+// compileViaYul: also
```

Comment:
Since this is the only test, perhaps we should add more tests with various bitwidths and assignment vs. other expressions